### PR TITLE
Added section - View Presenters on each platform

### PR DESCRIPTION
**Pull request by Daniel-Krzyczkowski**
## :thinking: Checklist before submitting

- [ ] All projects build
- [ ] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [ ] Nuspec files were updated (when applicable)
- [ ] Rebased onto current develop

### CODE REVIEW

File: docs/_documentation/fundamentals/view-presenters.md [hunk 1]
```diff
@@ -25,6 +25,43 @@ Let's take a look now at the methods of that interface:
...
+## View Presenters on each platform
+Each mobile platform has its own View Presenter:
+- Android: [MvxAndroidViewPresenter](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Droid/Droid/Views/MvxAndroidViewPresenter.cs)
+- iOS: [MvxIosViewPresenter](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs)
+- Windows: [MvxWindowsViewPresenter](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Windows/Uwp/Views/MvxWindowsViewPresenter.cs)
+
+When you navigate to selected ViewModel, platform specific View Presenter handles displaying View properly.
+
+<p align="center">
+  <img src="https://preview.ibb.co/cGuy8Q/View_Presenters.png">
```
Comment (MarcBruins) [inline docs/_documentation/fundamentals/view-presenters.md]: Can you add the image to the https://github.com/MvvmCross/MvvmCross/tree/develop/docs/assets/img folder?

Comment (Daniel-Krzyczkowski) [inline docs/_documentation/fundamentals/view-presenters.md]: Thanks Marc, of course but unfortunately I do not have permission to upload it. :(

Comment (Cheesebaron) [inline docs/_documentation/fundamentals/view-presenters.md]: Should be possible to add through a regular git client and not the GitHub web interface.

**Comment by Daniel-Krzyczkowski** [comment]
Thank you @Cheesebaron for hint - unfortunately there is the same problem - "You may not have permission to access the repository". Maybe I can just send this image to you and you can add it?

**Comment by MarcBruins** [comment]
You need to create a fork of the project. Add your image to the fork and add the reference in the markdown file. Checkout the tipcalc docs for a sample on how to reference the images. If you need more help you can ask in the MvvmCross slack channel or send a PM, i'd be happy to help out.

**Comment by Daniel-Krzyczkowski** [comment]
Thanks @MarcBruins - I did it as you said, please check now.

**Comment by MarcBruins** [comment]
Closed, this PR is continued in #2006

**Pull request closed without merging**